x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/constants/parameters_definition.ts

```diff
@@ -168,10 +175,48 @@ export const PARAMETERS_DEFINITION = {
       ],
     },
   },
+  null_value_numeric: {
+    fieldConfig: {
+      defaultValue: '',
+      label: nullValueLabel,
+      formatters: [toInt],
+    },
+  },
+  null_value_boolean: {
+    fieldConfig: {
+      defaultValue: '',
```

Comment:
I _think_ we should probably set a `defaultValue` here to one of the available options (e.g., `'true'`).

Reply:
Thanks for pointing this out! According to the docs:

> Accepts any of the true or false values listed above. The value is substituted for any explicit null values. Defaults to null, which means the field is treated as missing.

That last bit is what trips me up. It defaults to null, so maybe we need to add that as an option and default to that option? I will merge this PR now so we can build off it without introducing merge conflicts and then investigate and address all of the various null default values for the different data types in a subsequent PR.

Reply:
Sounds good!

Yeah, it is a little confusing. 

I don't think it's valid to set it to `null`.  If I try that, I get an error from ES: `Property [null_value] cannot be null.` I think the default behavior is for it not to be set. `boolean` is a little different from some of the other types that have this parameter in that it only excepts a set number of values, that is why we decided to make it a select instead of a text field. By setting `defaultValue` to an empty string, if the user enables the switch, but doesn't change the value from the dropdown, no value is sent back, even though it appears `"true"` is selected because it is the first item in the list of options. Maybe we should create an empty first option so the user is forced to make a selection?